apps/storybook/src/InformationPanel.stories.tsx

```diff
@@ -4,7 +4,7 @@
  *--------------------------------------------------------------------------------------------*/
 import { Story, Meta } from '@storybook/react';
 import React from 'react';
-import { CellProps } from 'react-table';
+import { TableTypes } from '@itwin/itwinui-react';
```

Comment:
this should use the new entrypoint?

Reply:
Fixed all the uses of Table's types outside of `packages/itwinui-react` that I could find.

https://github.com/iTwin/iTwinUI/pull/1514/files/a3edccd4c3ddc7f29e1f616a1def28e71780bfc0..809be55fa283f4ca0a1cd190aa031331ed2c1ceb